.circleci/config.yml

```diff
@@ -72,7 +72,7 @@ x-run:
       case "${TARGET_BRANCH}" in
           master|release-*)
               if git --no-pager diff --name-only HEAD "origin/${TARGET_BRANCH}" |
-                 grep -q -E '^(e2e|vendor|\.circleci)/'
+                 grep -q -E '^(go\.mod$|e2e/|\.circleci/)'
               then
                   # There are changes in .circleci, e2e or vendor
```

Comment:
nitpick that could now say `go.mod` not vendor... but the effect is on vendor eventually

Reply:
thanks!